Samples/excel-add-in-shape-dashboard/src/taskpane/taskpane.html

```diff
@@ -44,9 +44,9 @@
         <button id="change-format" class="ms-Button">
             <span class="ms-Button-label">Change information format</span>
         </button>
-        <p class="sampleSubTitle">Add event handler to dashboard. When dashbaord is activated/deactivated, the information will change.</p>
+        <p class="sampleSubTitle">Add onActivate/onDeactivated event handlers to the dashboard. These handlers will update the dashboard information when the dashboard is selected or unselected.</p>
```

Comment:
I finally got this to work. This is what I think it should say. 

```suggestion
        <p class="sampleSubTitle">While a cell outside the dashboard is selected, click the button to add onActivated/onDeactivated event handlers to the dashboard. Then click any shape on the dashboard and the type of information displayed changes. Then click any cell outside the dashboard and the information changes back. </p>
```